webpack-demo/src/tasks.js

```diff
@@ -0,0 +1,30 @@
+export const saveTasks = (tasks) => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+// Define the functions for adding tasks
+export const addTask = (description, tasks) => {
+  const task = {
+    description,
+    completed: false,
+    index: tasks.length + 1,
+  };
+  tasks.push(task);
+  saveTasks(tasks);
+  return task;
+};
+
+// Function for deleting or removing tasks
+export const deleteTask = (index, tasks) => {
+  tasks.splice(index, 1);
+  tasks.forEach((task, i) => {
+    task.index = i;
+  });
+  saveTasks(tasks);
+};
+
```

Comment:
- [ ] Great job so far. But there's one thing your need to work before every requirement of the project is met. According to the project requirement, after a todo is removed, the indexes of the remaining todos should be updated in a such a way that the task in the index of 3 should become task in the index of 2 if the task in the index of 2 has been deleted. Your app lacks this feature at this moment. Kindly fix it.


### Project Requirement
![project-requirements](https://user-images.githubusercontent.com/26597432/230594922-dc289a14-4c9e-482a-bc6d-e60ea9956b0e.png)


### Your App Behavior
[app-behave.webm](https://user-images.githubusercontent.com/26597432/230594860-1b2dd82d-a15d-4857-ac6f-5844a02fb6f3.webm)